assignment06/index.php

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>assignment06</title>
+    <meta charset="utf-8">
+</head>
+
+<body>
+    <!--
+            Allegra French
+            assignment06
+            INFO_2340_WW
+            Thoendel
+            1.26.2020
+        -->
+
+    <h1>Hello, world!</h1>
+    <div>Allow me to introduce myself! My name is <?php echo "Allegra French!"; ?></div>

```

Comment:
great php statement! Complete with quotation marks and that necessary semicolon! 👍